source/Note.hx

```diff
@@ -265,6 +265,18 @@ class Note extends FlxSprite
 				prevNote.updateHitbox();
 				// prevNote.setGraphicSize();
 			}
+			
+			switch(CoolUtil.difficultyString()) {
+				case 'Easy':
+					hitHealth = 0.06;
+					missHealth = 0.02;
+				case 'Normal':
+					hitHealth = 0.02;
+					missHealth = 0.04;
+				case 'Hard':
+					hitHealth = 0.01;
+					missHealth = 0.03;
```

Comment:
Don't know if this is intentional or not but is the missHealth supposed to be lower on Hard than Normal?

Reply:
Yep, it’s intentional. Press 3 notes to make up for that one miss.

Reply:
Alr then